config.js.example

```diff
@@ -11,8 +11,11 @@ const config = {
   // Your Bot's Token. Available on https://discordapp.com/developers/applications/me
   "token": "mfa.VkO_2G4Qv3T--NO--lWetW_tjND--TOKEN--QFTm6YGtzq9PH--4U--tG0",
 
-  // Default per-server settings. New guilds have these settings. 
+  "finnhub_api_key": "FINNHUB_KEY_HERE",
```

Comment:
Ooh good idea putting placeholders. I think they should go into config_base.js though (since it looks likes its used to write config.js, https://github.com/mynameisivan/discord-bot/blob/master/setup.js)

Reply:
Thanks. Pushed those changes already.

Reply:
Should we just get rid of this file then?

Reply:
@mynameisivan

Reply:
Yeah maybe its better to remove this file to reduce confusion. Can you do this as part of your branch?

Reply:
Yessir

Reply:
Done